components/automate-cli/cmd/chef-automate/automateHAAwsDeployment.go

```diff
@@ -30,7 +31,30 @@ func newAwsDeployemnt(configPath string) *awsDeployment {
 func (a *awsDeployment) doDeployWork(args []string) error {
 	if isA2HARBFileExist() {
 		err := executeDeployment(args)
-		return err
+		if err != nil {
+			return err
+		}
+		sharedConfigToml, err := getAwsHAConfig()
+		if err != nil {
+			return status.Wrap(err, status.ConfigError, "unable to fetch HA config")
+		}
+		archBytes, err := ioutil.ReadFile("/hab/a2_deploy_workspace/terraform/.tf_arch") // nosemgrep
```

Comment:
if you want to hardcode file locations these should be put in a constant

Reply:
Done